build.gradle

```diff
@@ -195,6 +195,7 @@ project(':ambry-account') {
                 project(':ambry-mysql')
         compile ("org.apache.helix:helix-core:$helixVersion") {
             exclude group: "log4j", module: "log4j"
+            exclude group: "org.codehaus.jackson"
```

Comment:
just to confirm, did you double check the version of jackson that is adopted by Helix lib?

Reply:
i remove this exclude statements. It causes some test failures.